app/etc/cas/config/cas.properties

```diff
@@ -1 +1,22 @@
-# Is overridden by the dogu
```

Comment:
Currently, we fill multiple .properties files:
* resources/etc/cas/config/cas.properties.tpl
* app/etc/cas/config/cas.properties
I think you should remove these additions as they are begin overwritten when cas starts. Two property files with the same entries makes it harder to find the file that actually introduce changes to the cas.

Reply:
The content in `app/etc/cas/config/cas.properties` is necessary for local development. So in my opinion we should keep it

Reply:
This is a good reason. I did not think of that.